src/features/ITGovernance/Requester/BusinessCase/AlternativeSolution/AlternativeSolutionFields.tsx

```diff
@@ -42,7 +61,8 @@ const AlternativeSolutionFields = ({
           error={!!flatErrors[`${altId}.title`]}
         >
           <Label htmlFor={`BusinessCase-${altId}Title`}>
-            {t('alternativeLabels.title', { altLabel })}
+            {t('solutionTitle')}
+            <RequiredAsterisk />
           </Label>
           <FieldErrorMsg>{flatErrors[`${altId}.title`]}</FieldErrorMsg>
           <Field
```

Comment:
```suggestion
          <Field
            className="maxw-none"
```

This field should be full width. It also needs to be updated in the Preferred Solution form.